src/main/java/org/sonarlint/intellij/issue/LiveIssue.java

```diff
@@ -208,6 +209,19 @@ public void setType(@Nullable String type) {
     return flows;
   }
 
+  public String getFlowsDescription() {
```

Comment:
Maybe this could be cached to avoid recomputing it every time the annotator runs?